src/Forms/Forms.Component.js

```diff
@@ -12,8 +12,8 @@ export const FormsComponent = () => {
             title='Forms'>
 
             <Example
-                description={`Inputs are used to collect data from the user. When a field is required, the label is displayed in bold
-                    and noted by an asterisk (*).`}
+                description={`Inputs are used to collect data from the user. When a field is required,
+                the <code>required</code> prop will include an asterisk (*).`}
```

Comment:
I think the playground components are setup to support markdown if you didn't want to hardcode markup tags.